i3xrocks.conf

```diff
@@ -97,3 +97,10 @@ interval=10
 [time]
 interval=60
 
+# Show title of focused window
+[focused-window-name]
```

Comment:
I would like to keep this disabled by default until/if we determine it is important enough that all users get this blocket by default.

Reply:
Wow, github has a lot of features I didn't know about :) Anyway, I pushed a commit that adds a bit more text in comments, but comments out the entire section.